test/performance/long-running-tests/README.md

```diff
@@ -0,0 +1,189 @@
+## Workload Benchmark for KubeStellar
+
+To generate the sample workload for KubeStellar performance experiments for long running tests proceed as following:
+
+1. Clone the following clusterloader2 repo. It's the tool used to deploy the benchmark workload in a KubeStellar environment:
+
+   ```bash 
+   git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
+
+   Set the variable `CL2_DIR` to the path of the subdirectory `clusterloader2/` of the cloned repo. For example: 
+
+   ```bash 
+   export CL2_DIR=$HOME/perf-tests/clusterloader2
+   ```
+   The script `test/performance/common/setup-clusterloader2.sh` uses the environment variable set above. 
+
+2. Configure clusterloader2 to generate KS performance workloads:
+
+   Starting from a local directory containing this KubeStellar git repo, run the following script to set-up your environment:
+
+   a) cd into the `test/performance/common` directory from your local copy of the KubeStellar repo, for example:
+
+   ```bash 
+   cd $HOME/kubestellar/test/performance/common
+   ```
+
+   b) set-up your environment:
+
+   ```bash 
+   ./setup-clusterloader2.sh --exp l
+   ```
+
+3. Configure the parameters of your workload:  
+
+   a) cd into the load configuration directory.
+
+   ```bash 
+   cd  $CL2_DIR/testing/load/
+   ```
+  
+   b) configure the parameters to create the workload traffic.
+   
+   ```bash 
+   vi long-duration-exp-config.yaml
+   ``` 
+
+   More specifically, configure the following parameter: 
+
+   - numberOfWorkloadObjects: total number of objects to be created in the experiment `perf-test` namespace (*default value: "10"*)
```

Comment:
Since the value in question is not a string, I would use code quotes rather than text quotes here (and also for the other numbers discussed below).

Reply:
ok, replaced with code quotes.